roles/openshift_logging_elasticsearch/tasks/restart_cluster.yml

```diff
@@ -65,6 +65,9 @@
       {{ openshift_client_binary }} get dc -l component={{ _cluster_component }},provider=openshift -n {{ openshift_logging_elasticsearch_namespace }} -o jsonpath={.items[*].metadata.name}
     register: _cluster_dcs
 
+  - set_fact:
+      _skip_healthcheck: "{{ __logging_scale_up | default(false) if _cluster_component == 'es' else __logging_ops_scale_up | default(false) }}"
```

Comment:
Can you explain this logic in a comment?  It seems to read as: "`_skip_healthcheck is assigned the value of `__logging_scale_up` if defined, else it defaults to `false` if the `_cluster_component` is `es` or `__logging_ops_scale_up` if defined, and if `__logging_ops_scale_up` is not defined that default to `false`."  Is that the right way to interpret this?

Reply:
correct, that's to interpret it. I can add a comment though so its more clear